cmd/k10s/main.go

```diff
@@ -63,17 +60,49 @@ func main() {
 		}
 	}
 
-	// Works even if client is nil or disconnected
+	pluginRegistry := plugins.NewRegistry()
+	pluginRegistry.Register(kitten.New())
+	log.Printf("Loaded %d plugins", len(pluginRegistry.List()))
+
 	log.Printf("Starting TUI...")
-	m := tui.New(cfg, client)
+	m := tui.New(cfg, client, pluginRegistry)
 
-	p := tea.NewProgram(
-		m,
-		tea.WithAltScreen(),
-		tea.WithMouseCellMotion(),
-	)
+	for {
+		p := tea.NewProgram(
+			m,
```

Comment:
nit: if you're going to reconstruct the model on each loop regardless you could inline the creation?
```suggestion
	for {
		p := tea.NewProgram(
			tui.New(cfg, client, pluginRegistry),
```

Reply:
Sure!